src/accountsdb/db.zig

```diff
@@ -255,12 +274,17 @@ pub const AccountsDB = struct {
         // used to merge thread results
         const n_combine_threads = n_threads;
 
+        var accounts_dir = try self.snapshot_dir.openDir("accounts", .{});
+        defer accounts_dir.close();
+
         var timer = std.time.Timer.start() catch unreachable;
         timer.reset();
 
         const n_account_files: usize = file_info_map.count();
         self.logger.infof("found {d} account files", .{n_account_files});
-        std.debug.assert(n_account_files > 0);
+
+        if (n_account_files == 0) return; // TODO: workaround for fuzz testing to work, not sure if correct behaviour
```

Comment:
probably want to add a warning log that there are no account files - this shouldnt happen during fuzz testing (or any valid snapshot) - somethings wrong with the file_info_map if it is

Reply:
Yeah good catch, I believe the reason this was actually happening is from a bug that I believe I incidentally fixed, I just forgot to go back and check if this was still needed. I'll check and see if it still causes issues, and add a warning if so, or remove this if not.

Reply:
9e03e90e5c99d4ea577099e38462ef4e04669ec4

Decided to just address it in the fuzz test itself - instead of checking in the loading function, we just don't start generating snapshot until we have a non-empty file map. Same outcome, simpler and more correct design.